ellama.el

```diff
@@ -176,11 +180,30 @@ ARGS contains keys for fine control.
 :buffer BUFFER -- BUFFER is the buffer (or `buffer-name') to insert ellama reply
 in.  Default value is (current-buffer).
 
-:point POINT -- POINT is the point in buffer to insert ellama reaply at."
+:point POINT -- POINT is the point in buffer to insert ellama reply at.
+
+:filter FILTER -- FILTER is a function applied to (partial) response strings.
+
+:session SESSION -- if session is non-nil, the PROMPT will be appended to the current (buffer-local)
```

Comment:
Might be nice to support multiple sessions per buffer in the future, but that's a future problem.